src/inversion_scripts/plumes.py

```diff
@@ -0,0 +1,753 @@
+import os
```

Comment:
Is there any filtering applied to plumes based on date? Or does it just take everything?

Reply:
I think the worry would be including point sources that are no longer present (eg. infrastructure fixes) from many years ago. I am not sure what the best solution is, so leaving it as is for now seems OK